build/llamafile-convert

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Please use `/bin/sh`. If you're using Linux then most systems should have that configured to be the Almquist Shell, which is a minimal POSIX compliant shell that's ultra fast.